internal/options/options.go

```diff
@@ -22,10 +22,15 @@ import (
 	"os"
 
 	"github.com/spf13/pflag"
+	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/klog"
 )
 
+var (
+	defaultEventTypes = []string{v1.EventTypeNormal, v1.EventTypeWarning}
```

Comment:
hmm what if new one gets added, should we not default to all instead of restricting? Or is there a reason for this?

Reply:
you are right, I will update the PR to default to all